### PR TITLE
luminous: tests: qa/rados/upgrade/jewel-x-singleton: set RBD_FEATURES

### PR DESCRIPTION
**Pull request by smithfarm**
https://tracker.ceph.com/issues/40249

### CODE REVIEW

File: qa/suites/rados/upgrade/jewel-x-singleton/5-workload/rbd_api.yaml [hunk 1]
```diff
@@ -8,4 +8,6 @@ split_tasks:
       clients:
         client.0:
            - rbd/test_librbd.sh
+      env:
+        RBD_FEATURES: 61
   - print: "**** done rbd/test_librbd.sh 7-workload"
```
Comment (yuriw) [inline qa/suites/rados/upgrade/jewel-x-singleton/5-workload/rbd_api.yaml]: This should be "print: "**** done rbd/test_librbd.sh 5-workload""

**Comment by yuriw** [comment]
[wip-yuri2-testing-2019-07-15-1956-luminous](https://trello.com/c/5QdUgJyi/667-wip-yuri2-testing-2019-07-15-1956-luminous)

**Comment by smithfarm** [comment]
@dillaman This PR seemed to cause the following failures:

* http://pulpito.ceph.com/yuriw-2019-07-24_14:09:38-rados-wip-yuri2-testing-2019-07-15-1956-luminous-2-distro-basic-smithi/4144810/
* http://pulpito.ceph.com/yuriw-2019-07-24_14:09:38-rados-wip-yuri2-testing-2019-07-15-1956-luminous-2-distro-basic-smithi/4144810/

Is the now-practically-EOL luminous worth looking into those, or should we just close this PR?

**Comment by dillaman** [comment]
@smithfarm It looks like those tests don't have logs anymore.

**Comment by smithfarm** [comment]
`teuthology-suite -k distro --priority 50 --suite rados/upgrade --email ncutler@suse.cz --ceph-repo https://github.com/ceph/ceph.git --ceph luminous --newest 10 --machine-type smithi --suite-repo https://github.com/smithfarm/ceph.git --suite-branch wip-40249`

**running** http://pulpito.ceph.com/smithfarm-2019-10-01_15:18:37-rados:upgrade-luminous-distro-basic-smithi/

**Comment by dillaman** [comment]
@smithfarm Try again after pushing the `jewel` branch to your forked `ceph` repo:

`
CommandFailedError: Command failed on smithi150 with status 128: 'rm -rf /home/ubuntu/cephtest/clone.client.0 && git clone --depth 1 --branch jewel https://github.com/smithfarm/ceph.git /home/ubuntu/cephtest/clone.client.0 && cd /home/ubuntu/cephtest/clone.client.0'`

**Comment by smithfarm** [comment]
@dillaman I already did that :-) New test is running now.

**Comment by smithfarm** [comment]
Teuthology run failed with:

```
Failure Reason:

'int' object is not iterable 
```

File: qa/suites/rados/upgrade/jewel-x-singleton/5-workload/rbd_api.yaml [hunk 2]
```diff
@@ -8,4 +8,6 @@ split_tasks:
       clients:
         client.0:
            - rbd/test_librbd.sh
-  - print: "**** done rbd/test_librbd.sh 7-workload"
+      env:
+        RBD_FEATURES: 61
```
Comment (dillaman) [inline qa/suites/rados/upgrade/jewel-x-singleton/5-workload/rbd_api.yaml]:
```suggestion
        RBD_FEATURES: "61"
```

Comment (smithfarm) [reply qa/suites/rados/upgrade/jewel-x-singleton/5-workload/rbd_api.yaml]: sigh...

**Comment by smithfarm** [comment]
**running** http://pulpito.ceph.com/smithfarm-2019-10-02_08:10:53-rados:upgrade-luminous-distro-basic-smithi/

**Comment by dillaman** [comment]
@smithfarm That fixes the teuthology issue. Now it looks like the test is core dumping.

**Comment by dillaman** [comment]
@smithfarm OK, so the new failure is due to running gmock tests under bionic. Jewel was never supported under bionic but now we are apparently building packages for it? This test should be restricted to jewel-supported releases.

**Comment by smithfarm** [comment]
**running** http://pulpito.ceph.com/smithfarm-2019-10-02_12:21:49-rados:upgrade-luminous-distro-basic-smithi/

**Comment by smithfarm** [comment]
With

```
  os_type: ubuntu
  os_version: '14.04'
```

I'm getting an ansible (provisioning) failure. I'm guessing Trusty is too old, and Xenial should be used instead.

**Comment by smithfarm** [comment]
**running** http://pulpito.ceph.com/smithfarm-2019-10-03_07:51:32-rados:upgrade-luminous-distro-basic-smithi/

**Comment by dillaman** [comment]
@smithfarm looks like both runs passed the RBD section. The CentOS one failed during cleanup because there was a ceph-osd coredump.

**Comment by smithfarm** [comment]
@yuriw Comment addressed. Please have another look?

**Comment by smithfarm** [comment]
@dillaman Thanks for your help.

**Pull request merged**